Xamarin.Forms.Platform.Android/Renderers/ContainerView.cs

```diff
@@ -62,13 +62,12 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 
 			var width = Context.FromPixels(r - l);
 			var height = Context.FromPixels(b - t);
-
-			_shellViewRenderer.LayoutView(width, height);
+			LayoutView(0, 0, width, height);
 		}
 
 		protected virtual void LayoutView(double x, double y, double width, double height)
 		{
-			View?.Layout(new Rectangle(x, y, width, height));
+			_shellViewRenderer.LayoutView(width, height);
```

Comment:
All this change does is makes sure that UpdateElevation is called when the header is created. This allows the user to set the flyout header elevation via the platform specific